### ISSUE TITLE
2.0.0-alpha.11 - Auth - A user with the repoReader Role on a repo testinput tries to run a put file -> wrong error msg

### ISSUE THREAD

**Issue by nadegepepin** [pachyderm/pachyderm#5959]
**What happened?**:
A user with the repoReader Role on a repo testinput tries to run a put file -> `needs permissions [REPO_INSPECT_COMMIT] on REPO testinput`

**What you expected to happen?**:
Was expecting the msg to be: `needs permissions [**REPO_WRITE**] on REPO testinput`

**Comment by nadegepepin** [comment]
- Create App and user1 in Auth0. 
- Create repo as Root user. 
- Give repoReader access to user1. 
- Connect as said user1 and attempt to add a file to the repo.

**Comment by nadegepepin** [comment]
NOT a bug. Works absolutely fine.  My mistake. 
I gave repoReader rights to a random email (instead of the right user) then logged in as the user who had not right on the repo and tried to write into it -> hence the basic REPO_INSPECT_COMMIT missing. 

Once I granted reader access to a repo to the user who tried to write in it: Got `<useremail> is not authorized to perform this operation - needs permissions [REPO_WRITE] on REPO testinput` which is the expected result.
